dotCMS/src/main/java/com/dotcms/rest/api/v1/portlet/PortletResource.java

```diff
@@ -120,4 +122,21 @@ public final Response deleteCustomPortlet(@Context final HttpServletRequest requ
     }
 
   }
+
+    @GET
+    @JSONP
+    @Path("/permissions/{portletId}")
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
```

Comment:
I see that this only returns a boolean, so I suggest doing: `/{portletId}/_doesuserhaveaccess`